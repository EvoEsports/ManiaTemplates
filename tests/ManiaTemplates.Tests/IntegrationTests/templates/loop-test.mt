<component>
    <using namespace="System.Linq"/>
    
    <import component="LoopComponent" as="LoopComponent" />
    
    <template>
        <LoopComponent foreach="int i in Enumerable.Range(1, 3)">
            <LoopComponent i="{{ i * 10 }}">
                <label text="outer_{{ i }}" />
            </LoopComponent>
        </LoopComponent>
    </template>
</component>