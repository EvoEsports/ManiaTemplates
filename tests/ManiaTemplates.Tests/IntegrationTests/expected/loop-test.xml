<manialink version="3" id="MtLoopTest" name="EvoSC#-MtLoopTest">
<label text="inner_-1" />
<label text="inner_10" />
<label text="outer_1" />
<label text="inner_-1" />
<label text="inner_20" />
<label text="outer_2" />
<label text="inner_-1" />
<label text="inner_30" />
<label text="outer_3" />
</manialink>
